ESLint-Formatter.py

```diff
@@ -7,6 +7,7 @@
 import glob
 import os, sys, subprocess, codecs, webbrowser
 from subprocess import Popen, PIPE
+from SublimeLinter.lint import persist
```

Comment:
Ah, my bad. That makes sense. For some reason I built up a mental model with SublimeLinter as a dep for the formatter. I removed it and added a debug flag to the settings. Hope that works.